examples/chef/chef.py

```diff
@@ -894,6 +897,13 @@ def main() -> int:
             else:
                 linux_args.append("chip_inet_config_enable_ipv4=false")
 
+            if options.enable_lit_icd:
+                linux_args.append("chip_enable_icd_server = true")
+                linux_args.append("chip_subscription_timeout_resumption = true")
```

Comment:
Is the persistent subscription feature enabled? It is a requirement for this feature.
As a side comment, this changes the behavior on when check-in messages are sent out based on wether we are retrying to re-subscribe or not.

Reply:
@mkardous-silabs `chip_persist_subscriptions` hasn't been enabled. Thanks for the reminder.  May I know what is your suggestion? 

Actually I am not very familiar with these feature. So I'll leave this to @yunhanw-google to comment if we should enable both `chip_persist_subscriptions` and `chip_subscription_timeout_resumption` .

Reply:
The ICDmanager behavior adapts based on the presence of both feature. The combos are
 
 - icd, without `chip_persist_subscriptions` and `chip_subscription_timeout_resumption`
 - icd, with `chip_persist_subscriptions` and without `chip_subscription_timeout_resumption`
 - all three
 
 Subscription timeout cannot be present without chip_persist_subscriptions IIRC. 
 
 making these configurable might be valuable to enable most tests case.